sphinx/screens/dashboard/dashboard/src/main/res/xml/motion_scene_dashboard.xml

```diff
@@ -15,6 +15,12 @@
     -->
     <ConstraintSet android:id="@+id/motion_scene_dashboard_drawer_closed">
 
+        <Constraint android:id="@id/layout_dashboard_popup">
+            <PropertySet
+                app:applyMotionScene="false"
```

Comment:
motion scenes should always contain the layout constraints as they're inherited in the `motion_scene_dashboard_drawer_opened` scene.